.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
   # Run the C benchmark tests. 
   c-benchmark-tests:
-    uses: lf-lang/benchmarks-lingua-franca/.github/workflows/benchmark-tests.yml@main
+    uses: lf-lang/benchmarks-lingua-franca/.github/workflows/benchmark-tests.yml@link-libraries
```

Comment:
This will only use a different version of the workflow definition, which hasn't changed, so this is not what you want to do.
I'm a little unsure as to what we do want to do; we're currently not using the workflow parameters, which are:
```
      compiler-ref:
        required: false
        type: string
      benchmarks-ref:
        required: false
        type: string
        default: main
```
I think we may want to specify `compiler-ref` to be `${{ github.ref }}`. I think it's fine not to specify `benchmarks-ref` and just test against `main`, but if you'd like to test against `link-libraries` you could do that. The problem is that after merging it needs to be reverted back to `main`...